core/input/input_event.h

```diff
@@ -445,6 +446,9 @@ class InputEventAction : public InputEvent {
 	void set_strength(float p_strength);
 	float get_strength() const;
 
+	void set_event_id(float p_id);
```

Comment:
```suggestion
	void set_event_id(int p_id);
```
I assume from the property